packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/shared/enums.ts

```diff
@@ -333,3 +434,16 @@ export enum DesyncMitigationMode {
    */
   STRICTEST = 'strictest',
 }
+
+/**
+ * Get the recommended TLS policy based on feature flags
+ * 
+ * @param scope The construct scope to check feature flags against
+ * @returns The appropriate SSL policy string
+ */
+export function getRecommendedTlsPolicy(scope: Construct): string {
+  if (FeatureFlags.of(scope).isEnabled(cxapi.ELB_USE_POST_QUANTUM_TLS_POLICY)) {
+    return SslPolicy.TLS13_12_RES_PQ;
```

Comment:
I think you should add a new `RECOMMENDED_TLS_PQ` enum to `SslPolicy`, and then change this line to be `return SslPolicy.RECOMMENDED_TLS_PQ`